automation/tests/authentication.spec.ts

```diff
@@ -1,17 +1,150 @@
+    await authenticationPage.createAccountButton.click();
+    await expect(authenticationPage.popupMessage).toBeVisible();
+    await expect(authenticationPage.popupMessage).toHaveText('Invalid email format!');
+  });
 
-})
\ No newline at end of file
+  test('Verify account cannot be created when username is missing', async ({page})=> {
+    await authenticationPage.navigateToRegistrationForm();
+    await authenticationPage.fillRegistrationForm(registrationData.USERNAME_MISSING, randomEmail, registrationData.PASSWORD_VALID, registrationData.PASSWORD_VALID);
+    await authenticationPage.createAccountButton.click();
+    await expect(authenticationPage.popupMessage).toBeVisible();
+    await expect(authenticationPage.popupMessage).toHaveText('You forgot to provide your username');
+  });
+
+  test('Verify account cannot be created when username is too short', async ({page})=> {
+    await authenticationPage.navigateToRegistrationForm();
+    await authenticationPage.fillRegistrationForm(registrationData.USERNAME_SHORT, randomEmail, registrationData.PASSWORD_VALID, registrationData.PASSWORD_VALID);
+    await authenticationPage.createAccountButton.click();
+    await expect(authenticationPage.popupMessage).toBeVisible();
+    await expect(authenticationPage.popupMessage).toHaveText('Your username has to have at least 5 characters');
+  });
+
+  test('Verify account cannot be created when passwords mismatch', async ({page})=> {
+    await authenticationPage.navigateToRegistrationForm();
+    await authenticationPage.fillRegistrationForm(randomString, randomEmail, registrationData.PASSWORD_VALID, registrationData.PASSWORD_SHORT);
+    await authenticationPage.createAccountButton.click();
+    await expect(authenticationPage.popupMessage).toBeVisible();
+    await expect(authenticationPage.popupMessage).toHaveText('Your passwords do not match!');
+  });
+  test('Verify account cannot be created when password is too short', async ({page})=> {
+    await authenticationPage.navigateToRegistrationForm();
+    await authenticationPage.fillRegistrationForm(randomString, randomEmail, registrationData.PASSWORD_SHORT, registrationData.PASSWORD_SHORT);
+    await authenticationPage.createAccountButton.click();
+    await expect(authenticationPage.popupMessage).toBeVisible();
+    await expect(authenticationPage.popupMessage).toHaveText('Your password must contain at least 6 symbols');
+  });
+});
+
+test.describe('@VB-32 - Logout', ()=> {
+    test('@Smoke - check logout function', async ({page})=> {
+    await authenticationPage.loginToAccount();
+    await expect(authenticationPage.profilePicture).toBeVisible();
+    await authenticationPage.profilePicture.click();
+    await expect(authenticationPage.profileDropdownMenu).toBeVisible();
+    await page.locator('.dropdownItem p:has-text("Logout")').click();
```

Comment:
Šis lokatorius turi būti page object modelyje. Testuose lokatorių nerašome

Reply:
ok, perkelsiu į POM